letsencrypt/reporter.py

```diff
@@ -16,6 +16,34 @@
 
 logger = logging.getLogger(__name__)
 
+def _get_terminal_size():
+    """Returns a list of terminal width and height"""
```

Comment:
For the indentation to work properly without wrapping the text, ``textwrap`` needs to be aware of the width of the terminal. Unfortunately passing 0 does not make it detect the width by itself, so I had to use that.